src/matcher/expression.js

```diff
@@ -18,9 +23,8 @@ const inBrowser = Boolean(typeof window !== 'undefined')
  * @return {string} Escaped string to allow converting into a regular expression
  */
 const escapeStr = str => {
-  // return inBrowser ? str.replace(/[.*+?^$()|[\]\\]/g, '\\$&') : str
   return inBrowser
-    ? str.replace(/[|\\()[\]^$+*?.]/g, '\\$&').replace(/-/g, '\\x2d')
+    ? str.replace(/[|\\[\]^$+*?.]/g, '\\$&').replace(/-/g, '\\x2d')
```

Comment:
I was meaning to do more investigations into this. I had a feeling is was not 100%
I added a `TODO` on line 196 of this file. Can you remove that, as it seems you've gotten it into a good spot.

Reply:
can do. Updated